src/rest.cls.php

```diff
@@ -359,6 +359,11 @@ public function is_rest($url = false)
 		// Debug2::debug( '[Util] is_rest check [base] ', $rest_url );
 		// Debug2::debug( '[Util] is_rest check [curr] ', $current_url );
 		// Debug2::debug( '[Util] is_rest check [curr2] ', wp_parse_url( add_query_arg( array( ) ) ) );
-		return strpos($current_url['path'], $rest_url['path']) === 0;
+		if($current_url!==false && isset($current_url['path']) && $rest_url!==false && isset($rest_url['path'])){
```

Comment:
Checking if its isset or not doesn't help much as using `strpos()` below, should check `empty()` or not.

Reply:
Done